docs/source/conf.py

```diff
@@ -0,0 +1,58 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'Pyseams'
+copyright = '2024, RuhiRG'
+author = 'RuhiRG'
+release = '0.1.0'
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+#referenced my favourite [1]
+extensions = [
+    "myst_parser",
+    "autodoc2",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.napoleon",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.todo",
+    "sphinx.ext.githubpages",
+    "sphinx_contributors",
+    "sphinx_copybutton",
+    "sphinx_design",
+    "sphinxcontrib.spelling",
+]
+
+autodoc2_render_plugin = "myst"
+autodoc2_packages = [
+    f"../../src/{project}",
+]
```

Comment:
```suggestion
autodoc2_render_plugin = "myst"
autodoc2_packages = [
    f"../../src/{project.lower()}",
]
```
Since the project is `Pyseams` but the directory is `pyseams`.